lib/gyp/generator/ninja/index.js

```diff
@@ -350,8 +353,8 @@ Ninja.prototype.actionCmd = function actionCmd(base, toBase, cmds) {
   if (this.flavor !== 'win32')
     return res;
 
-  // TODO(indutny): escape quotes in res
-  return `cmd.exe /s /c "${res}"`;
+  const wrap = gyp.platform.win.ninjaWrap;
+  return `${wrap} powershell -Command ${res.replace(/ & /g, ' ; ')}`;
```

Comment:
powershell is way more consistent with compound commands (e.g. {cd; xx.exe; copy xx})